CONTRIBUTING.md

```diff
@@ -0,0 +1,150 @@
+into an area where you feel more comfortable, and then perhaps into an area
+where you feel less comfortable or need to learn something new.
+
+### Doing the Work
+
+**tl;dr:**
+1. Work on a feature branch
+1. Run linting and tests often
+1. Ensure your change has test coverage (_ideally complete coverage_)
+1. Submit a descriptive pull request
+
+Once you have selected an issue and assigned it to yourself, you will start
+writing the code on an appropriately named branch on your local machine. If you
+are part of the Recidiviz Team, you may push branches directly to the
+`recidiviz/recidiviz.com` remote. Otherwise, you may push branches to a [fork](https://help.github.com/articles/fork-a-repo/)
+of the repository.
+
+Before issuing a pull request, run our linting and test suite over your changes
+to ensure no regression, as described in [our Readme](README.md).
+
+These same commands will be run by our continuous integration (CI) suite and
+your pull request will be "built against" automatically to show whether it
+passes. Stay ahead of the curve by running these throughout the development
+process to help you avoid any surprises.
+
+Related, include unit and//or integration tests to cover any new code. Update or
+remove existing test cases where appropriate.
+
+When you are passing linting and testing locally and are satisfied with the
+current state of your work, submit a pull request against `master`. However,
+you can also submit Work In Progress ("WIP") pull requests to gather early
+feedback if you believe it would be helpful. If you do so, please format the
+title as "WIP: My title here."
+
+Make sure the pull request description explicitly lists everything that the work
+does, and references any issues it contributes to.
+
+### Before Merging
+
+Before merging, all commits should be rebased and squashed down into a single
+atomic commit that does one thing. Squashing can be done either locally by hand
+or by an approver with the "Squash and Merge" button.
+
+If your pull request does two separate things, it should likely (but not
+necessarily) be multiple pull requests. We prefer for pull requests to be
+focused on one feature, issue, etc.
+
+We will not merge any pull requests that do not pass our CI suite, particularly
+those that do not pass linting, that do not pass all tests, or that avoidably
+lower our test coverage. (As noted above, our CI suite is run automatically for
+each new commit to each new pull request and will output the results on the pull
+request itself.) We will explicitly grant exceptions to this rule where
+appropriate. Modifying lint rules will not be granted without a well-reasoned
+justification.
+
+Once the above conditions are met, someone from the Recidiviz Team will merge
+your pull request (and delete the branch if issued against our remote).
+
+## Style
+
+Follow the [Airbnb Javascript Style Guide](https://github.com/airbnb/javascript).
```

Comment:
Follow the Airbnb Style Guides for [Javascript](https://github.com/airbnb/javascript) and [React](https://github.com/airbnb/javascript/tree/master/react).